packages/components/src/dimension-control/stories/index.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+/**
+ * Internal dependencies
+ */
+import { DimensionControl } from '..';
+import sizes from '../sizes';
+
+/**
+ * WordPress dependencies
+ */
+import { desktop, tablet, mobile } from '@wordpress/icons';
+
+export default {
+	component: DimensionControl,
+	title: 'Components (Experimental)/DimensionControl',
+	argTypes: {
+		onChange: { action: 'onChange' },
+		value: { control: { type: null } },
+		icon: {
+			control: { type: 'select' },
+			options: [ '-', 'desktop', 'tablet', 'mobile' ],
+			mapping: {
+				undefined,
```

Comment:
Nit: I believe that the keys of `mapping` need to map to the values in the `options` array. This is correct for `desktop`, `tablet` and `mobile`, but `'-'` needs to be mapped explicitly:

```suggestion
				[ '-' ]: undefined,
```